mappings/net/minecraft/server/world/ThreadedAnvilChunkStorage.mapping

```diff
@@ -62,7 +68,7 @@ CLASS net/minecraft/class_3898 net/minecraft/server/world/ThreadedAnvilChunkStor
 	METHOD method_17242 save (Z)V
 		ARG 1 flush
 	METHOD method_17244 updateHolderMap ()Z
-	METHOD method_17247 createEntityTickingChunkFuture (Lnet/minecraft/class_1923;)Ljava/util/concurrent/CompletableFuture;
+	METHOD method_17247 awaitEntitiesTickable (Lnet/minecraft/class_1923;)Ljava/util/concurrent/CompletableFuture;
```

Comment:
```suggestion
	METHOD method_17247 awaitTickableEntities (Lnet/minecraft/class_1923;)Ljava/util/concurrent/CompletableFuture;
```

Reply:
Hmm, this is more waiting for entities to be tickable, rather than waiting for tickable entities to appear